Dice-Game/GameEngine.cs

```diff
@@ -111,9 +111,22 @@ private void ChangeName()
         {
             string n;
             n = Console.ReadLine();
+
+            // Check out this cool cat function!!!!!
+            nul(n);
+
             Player.Name = n;
         }
 
+        // Any idiot would know what this does :LaughingFace:
+        private bool nul(string Name)

```

Comment:
This Function doesn't actually do anything if Name is null, just returns true or false. Nothing is actually done after the function is called either to change name if it is null.